docs/admin/authentication.md

```diff
@@ -52,6 +52,23 @@ optional group names.
 When using token authentication from an http client the apiserver expects an `Authorization`
 header with a value of `Bearer SOMETOKEN`.
 
+**Token Config File** is enabled by passing the `--token-config-file=SOMEFILE` option to the
+apiserver. The token config performs the same function as the above but is written in a yaml
+or json format (the decoder will attempt to automatically detect the format, though adding a
+extension is preferential). The file *MUST* contain token, name and uid fields with an optional
+groups array
+
+For example
+
+```JSON
+[
```

Comment:
All serialized formats must contain kind and apiVersion. Those that currently do not were a mistake.